### PR TITLE
Stop trying to make "Fetch" a thing.

### PR DESCRIPTION
**Pull request by TravisWhitaker**
This PR focuses on the Haskell data model, providing a stub definition of the 'Cacheable' monad. If the model doesn't offend anyone's sensibilities I'll commit the true 'Cacheable' implementation, including all implementations of 'Cacheable'-valued record fields.

### CODE REVIEW

File: src/CSH/Eval/Model.hs [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+    (==) = mkEq packetID
+
+instance Show Packet where
+    show p = intercalate "\n"
+        [ ("Packet ID: " ++ (show $ packetID p))
+        , ("Packet Due Date: " ++ (show $ packetDueDate p))
+        , ("Packet Percent Required: " ++ (show $ packetPercentReq p))
+        ]
+
+-- This should not be here...
```
Comment (gambogi) [inline src/CSH/Eval/Model.hs:280]: Where should it be?

**Comment by gambogi** [comment]
This looks fine to me

File: src/CSH/Eval/Model.hs (hunk 1 above)
Comment (TravisWhitaker) [inline src/CSH/Eval/Model.hs:280]: It should be under "Contexts." Back when we removed the queue table's ID, I neglected to update all the cache code to reflect it. I just noticed tonight.

Comment (harlanhaskins) [inline src/CSH/Eval/Model.hs:280]: Uh...

**Pull request merged**